cinder/templates/volumes/_vmware-configmap.yaml_tpl

```diff
@@ -0,0 +1,17 @@
+{{- define "volume_vmware_configmap" -}}
+{{- $context := index . 0 -}}
```

Comment:
It seems we are not using the global $context in this template so we could avoid the tuple foo in this case, no?